samples/udp/networking.js

```diff
@@ -107,7 +102,7 @@
       // Clients
       clients: {
         udp: {
```

Comment:
Since we don't use other types of clients, we may want to remove the "udp" key and simply call `var ec = new chromeNetworking.clients.echoClient();` in demo.js. What do you think?